common/utils/mission.js

```diff
@@ -132,6 +138,10 @@ export function computeMissionStats(m, users) {
     endTime,
     isComplete,
     validatedByAllMembers,
-    userStats
+    validatedAdminByAllMembers,
+    userStats,
+    adminValidation: m.validations
```

Comment:
Est-ce qu'on renommerait pas ici pour traduire le fait que c'est une validation globale ? Par exemple

```suggestion
    adminGlobalValidation: m.validations
```

Reply:
Oui j'ai hésité mais tu as raison. Je vais le faire.